src/main/resources/application.yaml

```diff
@@ -6,8 +6,8 @@ spring:
   datasource:
     driver-class-name: org.postgresql.Driver
     url: ${SPRING_DATASOURCE_URL}
-    password: ${SPRING_DATASOURSE_PASSWORD}
-    username: ${SPRING_DATASOURSE_USERNAME}
+    password: ${SPRING_DATASOURCE_PASSWORD}
+    username: ${SPRING_DATASOURCE_USERNAME}
```

Comment:
If you change it here, change these variables in docker-compose also.

Reply:
done